### PR TITLE
add Perl

### PR DESCRIPTION
**Pull request by jgreely**
Instructions for setting up both the Homebrew and Perlbrew versions of Perl.

### CODE REVIEW

File: Perl/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Homebrew depend on the version of Perl it installs, but for complete
+control of what version you're running and what packages are
+installed, there's also [Perlbrew](https://perlbrew.pl).
+
+## Installation
+
+### Homebrew
+
+```bash
+$ brew install perl cpanminus
```
Comment (simeg) [inline Perl/README.md]: A small explanation to why `cpanminus` is needed would be nice.

File: Perl/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+# revert to vendor/Homebrew Perl in current shell
+$ perlbrew off
+
+# reinstall all modules in a different version
+$ perlbrew clone-modules $from $to
+```
+
+## Usage
+
+To install additional modules from [CPAN](https://www.cpan.org):
```
Comment (simeg) [inline Perl/README.md]: Same here - what is CPAN?

File: Perl/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+To use Perlbrew in a shell script:
+
+```bash
+#!/bin/bash
+export PERLBREW_ROOT=/home/gugod/perl5/perlbrew
+export PERLBREW_HOME=/home/gugod/.perlbrew
+source ${PERLBREW_ROOT}/etc/bashrc
+perlbrew use perl-5.28.0
```
Comment (simeg) [inline Perl/README.md]: I don't understand fully understand L61-L-64. 
I think some more context can be added, answering questions like:
* Why do I need to declare these env variables?
* How do I find the `perlbrew` install location?
* Is `perlbrew use perl-5.28.0` a command?

**Comment by simeg** [comment]
@jgreely Hi,

Thanks for contributing! I left you a few comments.

File: Perl/README.md (hunk 3 above)
Comment (jgreely) [reply Perl/README.md]: I just removed that section. If you get far enough with Perl to need it, it's in the Perlbrew docs (in fact, directly copied from it).

File: Perl/README.md (hunk 1 above)
Comment (jgreely) [reply Perl/README.md]: Added brief explanation.

**Comment by simeg** [comment]
Looks good to me! 👍 

Would like it if someone else would approve and merge it.

@Kyslik @hugovk @sb2nov or anyone else

**Pull request merged**